src/test/java/org/jabref/preferences/VersionPreferencesTest.java

```diff
@@ -0,0 +1,21 @@
+package org.jabref.preferences;
+
+import org.jabref.logic.util.Version;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class VersionPreferencesTest {
+
+    @Test
+    public void givenNothingWhenCreatingThenNothingThrown() {
+        new VersionPreferences(Version.parse(""));
+    }
+
+    @Test
+    public void givenIgnoredVersionStringWhenGetIgnoredVersionThenUnknown() {
```

Comment:
In this test the system under test is the `VersionPreferences` class.
However, your assertion tests a very specific behaviour of the `Version` class. That's a mismatch for me.

Reply:
I would disagree with you about the mismatch. My current understanding of ```VersionPreferences``` leads me to think, that it is a specific subset of ```Version```. 
I cannot remember now, why I stumbled over ```getIgnoredVersion()```, but I used this base method to fiddle around with loading preferences. 
If the test does not belong in this class, where would you put it?